### PR TITLE
Create standard docsify navbar

### PR DESCRIPTION
**Pull request by grego952**
**Description**

Changes proposed in this pull request:

- Create standard docsify navbar

### CODE REVIEW

File: docs/_navbar.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+* <!-- markdown-link-check-disable -->
+* [Home](/README.md)
+* [Release Notes](/release-notes.md)
+* [Support & Contribution](/support-contribution.md)
+* <!-- markdown-link-check-enable -->
```
Comment (mmitoraj) [inline docs/_navbar.md:5]: Adding that comment creates an additional space in the UI that does not look good. Could try diable the check differently?

File: docs/_navbar.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+* <!-- markdown-link-check-disable -->
+* [Home](/README.md)
```
Comment (mmitoraj) [inline docs/_navbar.md:2]: I don't think it is necessary in the navbar.

File: docs/support-contribution.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Support and Contribution
```
Comment (mmitoraj) [inline docs/support-contribution.md:1]:
```suggestion
# Support & Contribution
```
To be consisted with what we display in the navbar.

File: docs/support-contribution.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+## Kyma Support
+
+Here are our support channels:
+
+- [Kyma Slack channels](http://slack.kyma-project.io/)
+- [Stack Overflow](https://stackoverflow.com/questions/tagged/kyma)
+- If you need to report a bug or have a feature request, raise an issue in the given repository using one of the available templates.
```
Comment (mmitoraj) [inline docs/support-contribution.md:9]:
```suggestion
- If you need to report a bug or have a feature request, raise an issue in the given module repository using one of the available templates.
```
Maybe we could also say that for general Kyma-related issues, the user should create an issue in the `/kyma` repo and add that link -> https://github.com/kyma-project/kyma/issues/new/choose (having a direct link is quite beneficial for the users I guess).

File: docs/support-contribution.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Support and Contribution
+
+## Kyma Support
```
Comment (mmitoraj) [inline docs/support-contribution.md:3]:
```suggestion
## Support
```
I'm not sure if "Kyma" is needed here.

File: docs/release-notes.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Release Notes
+
+As Kyma moved from the component-based to the module-based approach, all new versions of the Kyma modules are now released separately. The list below includes the GitHub release pages for all the available modules.
```
Comment (NHingerl) [inline docs/release-notes.md:3]:
```suggestion
As Kyma moved from the component-based to the module-based approach, all new versions of the Kyma modules are now released separately. 
See the GitHub release pages for all the available modules:
```

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**